### PR TITLE
Proposed change to inaugurator to test SSD IOPS

### PR DESCRIPTION
**Pull request by dror-lb**
1. installing fio in docker
2. add fio and dependencies
3. run fio test during H\W test and export it to selftest

Signed-off-by: Dror Vangosh <dror@lightbitslabs.com>

### CODE REVIEW

File: inaugurator/hwinfo.py [hunk 1]
```diff
@@ -163,6 +170,7 @@ def __init__(self):
     def run(self):
         data = {
                 "cpu": get_cpus(),
+                "fio": get_fio(),
```
Comment (abel-lb) [inline inaugurator/hwinfo.py]: assumption is that the lab will consume this info and fail is not as expectd ?

**Comment by yuval-lb** [comment]
@dror-lb what is the duration of the fio test

**Comment by dror-lb** [comment]
60 sec but we can reduce it to 10
Dror Vangosh
IT, Networking & DevOps
+972547782571

**Comment by yuval-lb** [comment]
> …

10 seconds is better, this test will run time we reclaim a machine.

File: inaugurator/hwinfo.py (hunk 1 above)
Comment (eranst-lb) [reply inaugurator/hwinfo.py]: yes

**Comment by eranst-lb** [comment]
> expose

selftest will handle this.

**Comment by eranst-lb** [comment]
> @dror-lb LGTM
> What are next steps, how do we plan to expose it to the lab so we know issue is exist?

selftest will handle this.

**Pull request merged**